parm/atm/obs/config/atms_n20.yaml

```diff
@@ -15,14 +15,16 @@ obs space:
 
 obs operator:
   name: CRTM
-  Absorbers: [H2O,O3]
+  Absorbers: [H2O, O3, CO2]
   Clouds: [Water, Ice]
   Cloud_Fraction: 1.0
   Cloud_Seeding: true 
   obs options:
     Sensor_ID: &Sensor_ID atms_n20
     EndianType: little_endian
     CoefficientPath: $(DATA)/crtm/
+  linear obs operator:
+    Absorbers: [H2O, O3]
```

Comment:
Hi Emily!  I was curious, do you mean to minimize J with respect to hydrometeors here?  If so, then the `Clouds: [Water, Ice]` section might need to be repeated in the `linear obs operator` below `Absorbers`.  BTW, thank you for putting in the effort, and also sharing all of these awesome YAML examples!  You have been a huge help.